### DISCUSSION TITLE
⚡️ Customer Support / Feature Requests ⚡️

### DISCUSSION THREAD

**Discussion by RTLS** [sequinstream/sequin#748] [Announcements]
Looking for support from the Sequin team? Or have a feature request?

The fastest way to talk with us (and the rest of the community) is in our discord, join here: https://discord.gg/8fRc7vSh

**Comment by Southclaws** [comment]
![image](https://github.com/user-attachments/assets/d527fb3e-738a-4526-936e-de51fc9b0cda)

Looks like the invite expired! (side note: thank you so much for choosing discord over yet another slack account I need to create 😂)

**Comment by zuzu-lulu** [comment]
does sequinstream work with clickhouse ? i see there is an example for tinybird but im confused if sequin works with clickhouse via kafka or nats ? maybe a tutorial would be great . i want to self host tinybird but looks like you have to register on the cloud and it doesn't support bare metal servers.
